### PR TITLE
docs: added Cheshire Cat in Integrations

### PR DESCRIPTION
**Pull request by nickprock**
Cheshire Cat is is a production-ready AI framework that uses Qdrant as vector memory, this PR is to be included in the list of Integrations.

New section preview: https://deploy-preview-419--condescending-goldwasser-91acf0.netlify.app/documentation/integrations/cheshire-cat/

@kacperlukawski @timvisee @Anush008

### CODE REVIEW

File: qdrant-landing/content/documentation/integrations/cheshire-cat.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+# Qdrant server
+# QDRANT_HOST=localhost
+# QDRANT_PORT=6333
+```
+
+Cheshire Cat takes great advantage of the following features of Qdrant:
+* [Collection Aliases](https://qdrant.tech/documentation/concepts/collections/#collection-aliases) to manage the change from one embedder to another.
+* [Quantization](https://qdrant.tech/documentation/guides/quantization/) to obtain a good balance between speed, memory usage and quality of the results.
+* [Snapshots](https://qdrant.tech/documentation/concepts/snapshots/) to not miss any information.
+* [Community](https://discord.com/invite/tdtYvXjC4h) and [documentation](https://qdrant.tech/documentation/)
```
Comment (timvisee) [inline qdrant-landing/content/documentation/integrations/cheshire-cat.md:38]: Please use local links for local resources, such as:

```diff
-* [Collection Aliases](https://qdrant.tech/documentation/concepts/collections/#collection-aliases) to manage the change from one embedder to another.
+* [Collection Aliases](../../concepts/collections/#collection-aliases) to manage the change from one embedder to another.

File: qdrant-landing/content/documentation/integrations/cheshire-cat.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## How to use the Cheshire Cat
+### Requirements
+To run the Cheshire Cat, you need to have [Docker](https://docs.docker.com/engine/install/) and [docker-compose](https://docs.docker.com/compose/install/) already installed on your system.
+
+### Starting the Cat
+For a quick start check the [instructions on github](https://github.com/cheshire-cat-ai/core/blob/main/README.md).
+
+### First configuration of the LLM
+
+* Open the Admin Portal in your browser at [localhost:1865/admin](localhost:1865/admin)
```
Comment (timvisee) [inline qdrant-landing/content/documentation/integrations/cheshire-cat.md:61]:
```suggestion
* Open the Admin Portal in your browser at [localhost:1865/admin](http://localhost:1865/admin)
```

**Review by timvisee — CHANGES_REQUESTED** [review]

File: qdrant-landing/content/documentation/integrations/cheshire-cat.md (hunk 1 above)
Comment (nickprock) [reply qdrant-landing/content/documentation/integrations/cheshire-cat.md]: Thanks, done.

File: qdrant-landing/content/documentation/integrations/cheshire-cat.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+<br>
+
+![RAG Pipeline](https://cheshirecat.ai/wp-content/uploads/2023/11/stregatto.jpg)
+
+<br>
+
+### Work in Progress
+We are working on a **separate retriever container** that use [fastembed](https://github.com/qdrant/fastembed) by default.
+
+For more information about our use of Qdrant check the articles about Vector Memory:
```
Comment (Anush008) [inline qdrant-landing/content/documentation/integrations/cheshire-cat.md:49]:
```suggestion
For more information about Cheshire Cat's use of Qdrant check the articles about Vector Memory:
```

File: qdrant-landing/content/documentation/integrations/cheshire-cat.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+For a quick start check the [instructions on github](https://github.com/cheshire-cat-ai/core/blob/main/README.md).
+
+### First configuration of the LLM
+
+* Open the Admin Portal in your browser at [localhost:1865/admin](http://localhost:1865/admin)
+* Configure the LLM in the `Settings` tab
+* If you don't explicitly choose it using `Settings` tab, the Embedder follows the LLM
+
+## Blog
+For more information check our [documentation](https://cheshire-cat-ai.github.io/docs/) and [blog](https://cheshirecat.ai/blog/), here three articles to get started with the Cheshire cat:
```
Comment (Anush008) [inline qdrant-landing/content/documentation/integrations/cheshire-cat.md:66]:
```suggestion
For more information refer to the Cheshire Cat [documentation](https://cheshire-cat-ai.github.io/docs/) and [blog](https://cheshirecat.ai/blog/).
```

File: qdrant-landing/content/documentation/integrations/cheshire-cat.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+* [Community](https://discord.com/invite/tdtYvXjC4h)
+
+<br>
+
+![RAG Pipeline](https://cheshirecat.ai/wp-content/uploads/2023/11/stregatto.jpg)
+
+<br>
+
+### Work in Progress
+We are working on a **separate retriever container** that use [fastembed](https://github.com/qdrant/fastembed) by default.
```
Comment (Anush008) [inline qdrant-landing/content/documentation/integrations/cheshire-cat.md:47]:
```suggestion
We are working on a **separate retriever container** that uses [fastembed](https://github.com/qdrant/fastembed) by default.
```

File: qdrant-landing/content/documentation/integrations/cheshire-cat.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+<br>
+
+## Cheshire Cat and Qdrant
+Cheshire Cat uses Qdrant as the default [Vector Memory](https://cheshire-cat-ai.github.io/docs/conceptual/memory/vector_memory/) for ingesting and retrieving documents.
+
+It uses:
+* memory based for testing
+* file based by default
+* container by settings two environment variables
```
Comment (Anush008) [inline qdrant-landing/content/documentation/integrations/cheshire-cat.md:22]:
```suggestion
```

File: qdrant-landing/content/documentation/integrations/cheshire-cat.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+* [Community](https://discord.com/invite/tdtYvXjC4h)
+
+<br>
+
+![RAG Pipeline](https://cheshirecat.ai/wp-content/uploads/2023/11/stregatto.jpg)
+
+<br>
+
+### Work in Progress
+We are working on a **separate retriever container** that uses [fastembed](https://github.com/qdrant/fastembed) by default.
```
Comment (Anush008) [inline qdrant-landing/content/documentation/integrations/cheshire-cat.md:42]:
```suggestion
Cheshire Cat is working on a **separate retriever container** that uses [fastembed](https://github.com/qdrant/fastembed) by default.
```

File: qdrant-landing/content/documentation/integrations/cheshire-cat.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+To run the Cheshire Cat, you need to have [Docker](https://docs.docker.com/engine/install/) and [docker-compose](https://docs.docker.com/compose/install/) already installed on your system.
+
+### Starting the Cat
+For a quick start check the [instructions on github](https://github.com/cheshire-cat-ai/core/blob/main/README.md).
+
+### First configuration of the LLM
+
+* Open the Admin Portal in your browser at [localhost:1865/admin](http://localhost:1865/admin)
+* Configure the LLM in the `Settings` tab
+* If you don't explicitly choose it using `Settings` tab, the Embedder follows the LLM
```
Comment (Anush008) [inline qdrant-landing/content/documentation/integrations/cheshire-cat.md:58]:
```suggestion
* Open the Admin Portal in your browser at [localhost:1865/admin](http://localhost:1865/admin).
* Configure the LLM in the `Settings` tab.
* If you don't explicitly choose it using `Settings` tab, the Embedder follows the LLM.
```

**Review by Anush008 — APPROVED** [review]
Very cool ser👌.

**Comment by nickprock** [comment]
Thanks @Anush008 :pray:

File: qdrant-landing/content/documentation/integrations/cheshire-cat.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+CORE_HOST=localhost
+CORE_PORT=1865
+
+# Qdrant server
+# QDRANT_HOST=localhost
+# QDRANT_PORT=6333
+```
+
+Cheshire Cat takes great advantage of the following features of Qdrant:
+* [Collection Aliases](../..//concepts/collections/#collection-aliases) to manage the change from one embedder to another.
```
Comment (timvisee) [inline qdrant-landing/content/documentation/integrations/cheshire-cat.md:30]:
```suggestion
* [Collection Aliases](../../concepts/collections/#collection-aliases) to manage the change from one embedder to another.
```

Comment (nickprock) [reply qdrant-landing/content/documentation/integrations/cheshire-cat.md]: Thanks, done

**Review by timvisee — APPROVED** [review]

**Comment by nickprock** [comment]
Hi @Anush008 @timvisee @kacperlukawski 
Is there anything else that needs to be done to be merged?

File: qdrant-landing/content/documentation/integrations/cheshire-cat.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+Cheshire Cat takes great advantage of the following features of Qdrant:
+* [Collection Aliases](../../concepts/collections/#collection-aliases) to manage the change from one embedder to another.
+* [Quantization](../../guides/quantization/) to obtain a good balance between speed, memory usage and quality of the results.
+* [Snapshots](../../concepts/snapshots/) to not miss any information.
+* [Community](https://discord.com/invite/tdtYvXjC4h)
+
+<br>
+
+![RAG Pipeline](https://cheshirecat.ai/wp-content/uploads/2023/11/stregatto.jpg)
```
[the remark this answers is not in the document]
Comment (nickprock) [reply qdrant-landing/content/documentation/integrations/cheshire-cat.md]: Done

File: qdrant-landing/content/documentation/integrations/cheshire-cat.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+weight: 600
+---
+
+# Cheshire Cat
+
+[Cheshire Cat](https://cheshirecat.ai/) is an open-source framework that allows you to develop intelligent agents on top of many Large Language Models (LLM). You can develop your custom AI architecture to assist you in a wide range of tasks.
+
+<br>
+
+![Cheshire cat](https://cheshirecat.ai/wp-content/uploads/2023/11/cat.jpg)
```
[the remark this answers is not in the document]
Comment (nickprock) [reply qdrant-landing/content/documentation/integrations/cheshire-cat.md]: Done

**Comment by kacperlukawski** [comment]
@nickprock I'll merge the PR soon, but I will fix the images on my own. Thanks for the contribution!

**Review by kacperlukawski — APPROVED** [review]

**Comment by nickprock** [comment]
Thanks @kacperlukawski @Anush008 @timvisee I'll disturb you at the next update

**Pull request merged**